airflow/contrib/operators/gcs_to_gcs.py

```diff
@@ -112,6 +118,8 @@ def __init__(self,
                  destination_bucket=None,
                  destination_object=None,
                  move_object=False,
+                 filter_by_last_modified=False,
```

Comment:
Please move the new arguments to the end of the list, to maintain backward compatibility.